### PR TITLE
Feature/zen 27285 add filter support

### PR DESCRIPTION
**Pull request by mozart27**
This commit adds the the ability to provide a filter in a CentralQuery query.

### CODE REVIEW

**Comment by jpeacock-zenoss** [comment]
Does this change effect the use of tags by existing code?

**Comment by mozart27** [comment]
@jpeacock-zenoss - no.  Any code using tags will not be affected by this change.

File: central-query-lib/src/main/java/org/zenoss/app/metricservice/api/model/v2/MetricQuery.java [hunk 1]
```diff
@@ -14,11 +14,11 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
```
Comment (jpeacock-zenoss) [inline central-query-lib/src/main/java/org/zenoss/app/metricservice/api/model/v2/MetricQuery.java]: (nitpick) Why was this line moved from below.

Comment (mozart27) [inline central-query-lib/src/main/java/org/zenoss/app/metricservice/api/model/v2/MetricQuery.java]: I think I removed it and re-added it.  I'm going to move it back since it's in the wrong place now.
